aerogear/aerogear-unifiedpush-server
# Aerogear 9897

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9897 

This is it, the PR that let's you send notifications to browsers.  Right now there's still a lot of work left to do.

## How
This adds 2 new queues in all of the right places, adds handling for web push variants to the messaging system, and actually sends messages.  It is missing a few things; most notably lifecycle management for messages.

## Verification Steps
 * Setup UPS with a web push variant (see recent previous PRs)
 * Register a client with UPS (please use the js-sdk PR here : https://github.com/aerogear/aerogear-js-sdk/pull/431)
 * Send a message
 * Ensure message is fetched

## Progress

- [x] Messages send


## Notes
I removed the "check extra properties" progress task.  This is probably better handled in a different PR as the notification generated by the JS SDK needs to support those as well and we need to figure out what properties we want to support and what they look like.
